src/SearchParameters/index.js

```diff
@@ -1066,6 +1066,9 @@ SearchParameters.prototype = {
     if (this.isHierarchicalFacetRefined(facet)) {
       throw new Error(facet + ' is already refined.');
     }
+    if (!this.isHierarchicalFacet(facet)) {
+      throw new Error(facet + ' is not defined in the hierarchicalFacets attribute of the helper configuration.');
```

Comment:
We could add a test to cover this case.

Reply:
there's already a test to cover this (the one which failed in the first commit)